packages/devtools_app/lib/src/service_extensions.dart

```diff
@@ -96,44 +101,68 @@ final debugAllowBanner = ToggleableServiceExtensionDescription<bool>._(
 
 final invertOversizedImages = ToggleableServiceExtensionDescription<bool>._(
   extension: 'ext.flutter.invertOversizedImages',
-  description: 'Invert Oversized Images',
-  icon: const Icon(Icons.image, size: actionsIconSize),
+  description: 'Highlight Oversized Images',
+  disabledIcon: const ThemedImageIcon(
+    lightModeAsset: 'icons/images-dgrey.png',
+    darkModeAsset: 'icons/images-lgrey.png',
+  ),
+  enabledIcon: const ThemedImageIcon(
+    lightModeAsset: 'icons/images-white.png',
+    darkModeAsset: 'icons/images-dgrey.png',
+  ),
   enabledValue: true,
   disabledValue: false,
-  enabledTooltip: 'Disable Invert Oversized Images',
-  disabledTooltip: 'Enable Invert Oversized Images',
+  enabledTooltip: 'Disable Highlight Oversized Images',
+  disabledTooltip: 'Enable Highlight Oversized Images',
   gaScreenName: analytics_constants.inspector,
   gaItem: analytics_constants.debugBanner,
 );
 
 final debugPaint = ToggleableServiceExtensionDescription<bool>._(
   extension: 'ext.flutter.debugPaint',
-  description: 'Debug Paint',
-  icon: createImageIcon('icons/debug_paint@2x.png'),
+  description: 'Show Guidelines',
+  disabledIcon: const ThemedImageIcon(
+    lightModeAsset: 'icons/guidelines-dgrey.png',
+    darkModeAsset: 'icons/guidelines-lgrey.png',
+  ),
+  enabledIcon: const ThemedImageIcon(
+    lightModeAsset: 'icons/guidelines-white.png',
+    darkModeAsset: 'icons/guidelines-dgrey.png',
+  ),
   enabledValue: true,
   disabledValue: false,
-  enabledTooltip: 'Hide Debug Paint',
-  disabledTooltip: 'Show Debug Paint',
+  enabledTooltip: 'Hide Guidelines',
+  disabledTooltip: 'Show Guidelines',
   gaScreenName: analytics_constants.inspector,
   gaItem: analytics_constants.debugPaint,
 );
 
 final debugPaintBaselines = ToggleableServiceExtensionDescription<bool>._(
   extension: 'ext.flutter.debugPaintBaselinesEnabled',
-  description: 'Paint Baselines',
-  icon: createImageIcon('icons/inspector/textArea@2x.png'),
+  description: 'Show Baselines',
+  disabledIcon: const ThemedImageIcon(
+    lightModeAsset: 'icons/baselines-dgrey.png',
+    darkModeAsset: 'icons/baselines-lgrey.png',
+  ),
+  enabledIcon: const ThemedImageIcon(
+    lightModeAsset: 'icons/baselines-white.png',
+    darkModeAsset: 'icons/baselines-dgrey.png',
+  ),
   enabledValue: true,
   disabledValue: false,
-  enabledTooltip: 'Hide Paint Baselines',
-  disabledTooltip: 'Show Paint Baselines',
+  enabledTooltip: 'Hide Baselines',
```

Comment:
We should add a TODO in this file to add richer tooltips, as described in https://docs.google.com/document/d/1xBJpVykI7-UYP61MX6bKZMl6awQwNvEc3jurDjSZEVE/edit?ts=60db89a6#heading=h.ytka7us6qnox. If we don't already have an issue in the repo to track this, we should also create one and link to it in the TODO

Reply:
done